src/client/opamArg.ml

```diff
@@ -23,6 +23,14 @@ let when_enum =
   [ "always", `Always; "never", `Never; "auto", `Auto ]
   |> List.map (fun (s,v) -> cli_original, s, v)
 
+let confirm_enum =
+  [
+    cli_from cli2_1, "ask", `ask;
+    cli_from cli2_1, "yes", `all_yes;
+    cli_from cli2_1, "no", `all_no;
```

Comment:
```suggestion
    cli_from cli2_1, "no", `all_no;
    cli_from cli2_1, "yes", `all_yes;
```
Cosmetically, it means the orders are decreasing order of safety:
- Ask user (i.e. no automatic answer)
- No
- Yes
- Unsafe-yes